app/containers/LocationPage/index.js

```diff
@@ -120,16 +121,22 @@ export class LocationPage extends React.Component { // eslint-disable-line react
                 </PageSection>
 
                 <PageSection title="Gear List">
-                  <BorderBottomDiv>
-                    Bring climbing shoes or something
-                  </BorderBottomDiv>
+                  <GearListCard gearListObj={
```

Comment:
Let's define this props object outside the render and pass it in

Reply:
I put it outside the return, next to the metadata array. Is that what you meant?